scripts/util.py

```diff
@@ -207,25 +207,30 @@ def get_credentials() -> Optional[service_account.Credentials]:
 
 def create_client_options(api: ApiEndpoint = None) -> ClientOptions:
     """Create client options for cloud endpoints"""
-    ep = None
     ver = endpoint_version(api)
     ud = universe_domain()
-    if ud == DEFAULT_UNIVERSE_DOMAIN:
-        ud = None
+    co = ClientOptions()
+    if ud and ud != DEFAULT_UNIVERSE_DOMAIN:
+        try:
+            co.universe_domain = ud
+        except Exception:
+            log.error(
+                "Universe domain is not part of ClientOptions with installed version of google-api-core."
+            )
+            exit(1)
```

Comment:
This function should have no business to decide killing the process, just raise the exception.

Reply:
Done